packages/IMAPClient-UI.package/ICLoginDialog.class/instance/emptyParametersIn..st

```diff
@@ -0,0 +1,4 @@
+helper
+emptyParametersIn: aDictionary
+
+	^ (aDictionary select: [ :value | (value asString isEmpty) | (value = nil asString) ]) keys
```

Comment:
Not sure how our UI works, but the check `value = nil asString` might have this unintended sideefect:
A user, who wants his name to be 'nil' will get an Error Message, saying that this parameter was empty.

Is there the need to check for `nil asString`? Does our UI create the String 'nil' (which is the result of `nil asString`, uninitialized Strings are just empty)? Why is this check necessary?

Reply:
The UI call for the form contents gives us Strings, no matter what. Therefore this `nil` would have to be sorted out elsewhere. However, I think this edge case behaviour would be reasonable in favor of having all of these checks in one location.

Reply:
I would argue, that the UI would never give you a "nil"-String.
But alright, gonna do an issue for that later